podcasts/ThemeableSelectionView.swift

```diff
@@ -65,4 +68,13 @@ class ThemeableSelectionView: UIView {
         backgroundColor = AppTheme.colorForStyle(style, themeOverride: themeOverride)
         layer.borderColor = isSelected ? AppTheme.colorForStyle(selectedStyle, themeOverride: themeOverride).cgColor : AppTheme.colorForStyle(unselectedStyle, themeOverride: themeOverride).cgColor
     }
+
+    private func updateAccessibilityLabel() {
+        if unselectedAccessibilityLabel == nil {
+            unselectedAccessibilityLabel = accessibilityLabel
+        }
+        let unselectedAccessibilityLabel = self.unselectedAccessibilityLabel ?? ""
+        let selectedAccessibilityLabel = "\(L10n.statusSelected), \(unselectedAccessibilityLabel)"
+        accessibilityLabel = isSelected ? selectedAccessibilityLabel : unselectedAccessibilityLabel
```

Comment:
I think we can achieve the same result in a bit cleaner fashion by using the accessibilityValue and adding a new string for statusUnselected "Unselected".

You can view the docs for adding a new string here: https://github.com/Automattic/pocket-casts-ios/blob/trunk/docs/localization.md

```suggestion
        accessibilityValue = isSelected ? L10n.statusSelected : L10n.statusUnselected
```

Reply:
@emilylaguna  thank you so much for suggesting to use accessibilityValue. I did not add new string for unselected as that is not needed because it is enough to indicate when something is selected. In fact, this is how Apple's controls work. Try setting isSelected to true on UIButton and word selected will be pronounced when button is selected. However, when button is not selected it wont pronounce unselected but will just read its accessibility label.

Reply:
Ah, I see. I think in that case we can stay consistent with how other buttons work and use the selected property on [UIAccessibilityTraits](https://developer.apple.com/documentation/uikit/uiaccessibilitytraits).

```
if isSelected {
    accessibilityTraits.insert(.selected)
} else {
    accessibilityTraits.remove(.selected)
}
```

wdyt?

Reply:
Wow, this is even better.